plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FlexFacetUninstallDelegate.java

```diff
@@ -0,0 +1,17 @@
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.wst.common.project.facet.core.IDelegate;
+import org.eclipse.wst.common.project.facet.core.IProjectFacetVersion;
+
+public class FlexFacetUninstallDelegate implements IDelegate {
+
+  @Override
+  public void execute(IProject project, IProjectFacetVersion fv, Object config, IProgressMonitor monitor)
+      throws CoreException {
+    
```

Comment:
We decided not to do that since users can do deploy with a project that doesn't have the app.yaml because gcloud would create one on its own.